app/src/main/java/com/epam/themes/androidcomponents/SimpleFragment.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.themes.androidcomponents;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.epam.cleancodetest.R;
+
+public class SimpleFragment extends Fragment {
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull final LayoutInflater inflater, @Nullable final ViewGroup container, @Nullable final Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_simple, container, false);
+    }
+
+    @Override
+    public void onSaveInstanceState(@NonNull final Bundle outState) {
```

Comment:
this and 2 methods below is empty, please remove.
In HW you need to write saving state logic, not just override methods